certified-connectors/PowerBI Robots/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Account Id",
+          "description": "The Account Id for this api",
+          "tooltip": "Provide your Account Id",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0577bd",
+    "capabilities": [],
+    "publisher": "Ant\u00f3nio Arantes"
```

Comment:
@AntonioArantesDevscope Please add 'stackOwner' property as well.

Reply:
commited on Identation, response and missing properties corrections